packages/frontend-app/tailwind.config.js

```diff
@@ -49,10 +54,22 @@ module.exports = {
         mediumGray: "#ABA6A6",
       },
 
+      borderRadius: {
+        50: "50%",
+      },
+
       zIndex: {
         1000: "1000",
       },
 
+      spacing: {
+        9: "9px",
+        144: "144px",
+        156: "156px",
+        214: "214px",
+        250: "250px",
+      },
```

Comment:
I don't think we need any of these. Please remove. Use the closest unit from tailwind that you can find to these values